x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/job/process/autodetect/state/ModelSizeStatsTests.java

```diff
@@ -22,6 +22,8 @@
     public void testDefaultConstructor() {
         ModelSizeStats stats = new ModelSizeStats.Builder("foo").build();
         assertEquals(0, stats.getModelBytes());
+        assertEquals(0, stats.getModelBytesExceeded());
+        assertEquals(0, stats.getModelBytesMemoryLimit());
```

Comment:
These two will change to `null` if the other changes I recommended are made.  So that makes the values set by the default constructed builder inconsistent.  But this is better than filling in values that didn't really exist in a JSON document.